### PR TITLE
feat: add new members

### PR DESCRIPTION
**Pull request by momo296**
Update the members' page.

## Description

Update the members' page.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by YHhaoareyou** [comment]
I just rebased it to `develop`

**Review by nichnarmada — CHANGES_REQUESTED** [review]
Please check my comments!

File: root/src/components/aboutUs/MeetOurTeam/MeetOurTeam.tsx [hunk 1]
```diff
@@ -31,7 +42,7 @@ const Card = styled("div")`
   display: flex;
   flex-direction: column;
   align-items: center;
-  margin: 40px 20px 20px 0px;
+  margin: 40px 30px 30px;
```
[the remark this answers is not in the document]
Comment (momo296) [reply root/src/components/aboutUs/MeetOurTeam/MeetOurTeam.tsx:45]: Sure!

**Review by nichnarmada — APPROVED** [review]

**Review by YHhaoareyou — APPROVED** [review]
LGTM

**Pull request merged**